docs/src/docker.md

```diff
@@ -165,3 +164,18 @@ The image will be tagged as `playwright:localbuild-jammy` and could be run as:
 docker run --rm -it playwright:localbuild /bin/bash
 ```
 
+## Development
+* langs: python
+
+### Build the image
```

Comment:
Why do we need this section? Our main recommendation is our image, other enthusiasts can figure it out themselves.

Reply:
we have a similar section, especially the:

```Dockerfile
FROM python:3.12-bookworm

RUN pip install playwright==@%%VERSION%% && \
    playwright install --with-deps
```

part for Node.js since https://github.com/microsoft/playwright/pull/28434 - for Python I thought it makes sense as well, since users often create their own / have their own image.

Reply:
My question is about the section in general regardless the language. This will just confuse people new to docker.